module-2/newspaper/index.html

```diff
@@ -5,61 +5,71 @@
+                    <div> <img src="https://www.theadvertiser.com/gcdn/presto/2022/04/12/PSHR/2580dc8b-d28d-4d65-925d-edefa2996413-66.jpg?crop=5807,3267,x0,y441&width=210&height=118&format=pjpg&auto=webp" alt="Louisiana Senator" width="90%"><figcaption>Senator Bill Cassidy speaks to Louisiana Boys State</figcaption></div>
+                    <div> <img src="https://www.theadvertiser.com/gcdn/media/2021/06/12/SouthBend/0075fb2bf01ca486cf8beef48296a476.jpg?crop=800,450,x0,y62&width=210&height=118&format=pjpg&auto=webp" alt="Now Hiring" width="90%"><figcaption>61,000 Louisiana workers quite in April, data shows</figcaption></div>
+                    <div> <img src="https://www.theadvertiser.com/gcdn/presto/2023/06/27/PLAL/1ae2648f-9f03-4a8f-963b-55db184d96f9-USATSI_20954334.jpg?crop=4685,2636,x0,y238&width=210&height=118&format=pjpg&auto=webp" alt="Champions" width="90%"><figcaption>A way-too-early preview of LSU baseball's 2024 roster</figcaption></div>
                 </div>
-                <div class="flex-container4">
-                    <div> Senator Bill Cassidy speaks to Louisiana Boys State</div>
-                    <div> 61,000 Louisiana workers quite in April, data shows  </div>
-                    <div> A way-too-early preview of LSU baseball's 2024 roster </div>
+                <div class="watch-now">
+                    <h2>Watch Now__________________________________________</h2>
+                    <video controls autoplay muted loop width="675" height="350" >
+                        <source src="https://joy1.videvo.net/videvo_files/video/free/2019-11/large_watermarked/190301_1_25_11_preview.mp4" type="video/mp4">
+                        <a href="https://joy1.videvo.net/videvo_files/video/free/2019-11/large_watermarked/190301_1_25_11_preview.mp4"></a>
+                    </video>
+                    <h4>Husbands try to keep balance in the annual wife-carrying race filled with obstacles</h4>
+                    <div>Husbands competed in the annual wife-carrying contest, attempting to stay balanced through the obstacles with their ladies on their shoulders. <em>Ariana Triggs, TMX Video</em></div>
                 </div>
             </div>
             <div class="right-column">
-        <div class="top-headlines">
-            <h2><strong>Top Headlines_______</strong></h2>
-            <div>BJ's Pizza is closing,but Acadian shows support online and in person</div>
-            <div>How LSU baseball coalesced into a national championship-caliber roster</div>
-            <div>LSU baseball fans celebrates national title. Fans say its worth the wait</div>
-            <div>PHOTOS-LSU Baseball Championship Celebration</div>
-        </div>
-        <div class="improvements">
-            <h4><em>We're always working to improve your experience. <u> Let us know what you think.</u></em></h4>
-        </div>
+                <div class="top-headlines">
+                    <h2><strong>Top Headlines___________</strong></h2>
```

Comment:
In general you'll want to style text using CSS (e.g.: 'font-weight: bold' or 'font-weight': 900) as opposed to using `strong` and related styled HTML elements. 

Also, things like the `____________` line after "Headlines" wouldn't typically be done in HTML like this. Instead, you'd place a `div` next to the `h2` and style it using a "border-bottom" property to give the effect of a line. I'd strongly encourage you to try this out for yourself!